api/src/main/java/com/messagebird/objects/Request.java

```diff
@@ -28,15 +32,28 @@ public Request(String timestamp, String queryParameters, byte[] data) {
         this.data = data;
     }
 
-    String getTimestamp() {
+    public String getTimestamp() {
```

Comment:
I suggest not exposing these as part of the public API and keeping them package private instead. IMHO, doing this only adds the burden of having to remain backwards compatible without adding any benefit

Reply:
(Also goes for other methods on the Request class).

Reply:
good point!